TSC-Charter.md

```diff
@@ -216,6 +225,15 @@ TSC.
 * **Maintainer**: a Collaborator within a Core Project elected to
 represent the Core Project on the TSC.
 
+## Section 11. Escalation of Disputes and Code of Conduct Violations
+
+Participation in the Node.js project is governed by a Code of Conduct policy
+that is established by the TSC. Any accusations of Code of Conduct or policy
```

Comment:
We end up in the same boat with the same hand wringing, delays and interpersonal issues. Going immediately to mediation is better. It doesn't have to be involved. This essentially just means that the Foundation picks someone to look at the issue and decide.